networkd/network_server.ml

```diff
@@ -839,11 +839,37 @@ module Bridge = struct
 		Debug.with_thread_associated dbg (fun () ->
 			match !backend_kind with
 			| Openvswitch ->
-				Ovs.bridge_to_interfaces name
+				Ovs.get_real_bridge name
+				|> Ovs.bridge_to_interfaces
```

Comment:
I didn't mean to change this function. This is wrong, because it changes the semantics. Instead, the function `add_pvs_proxy_port` has some lines that can be replaced by the new function.